### PR TITLE
Bugfix/play icon not visible

### PR DESCRIPTION
**Pull request by l-suzuki**
This PR responds to: 
CC-407 https://rabidtech.atlassian.net/browse/CC-407 
Zendesk ticket #798 https://ackama.zendesk.com/agent/tickets/798 

The issue: NZSL had reports from Android users that the ‘play’ button has disappeared from the videos in both the search results and the sign pages when viewed on a mobile phone. The video still works when you click on it, but users don’t know to click on the video when there is no play button.

This only affects the responsive view of the website on Android (i.e. the full website view, Android mobile app and iOS mobile app are all fine)

There was a line in buttons.scss that was disabling the play button explicitly for the above use case. This has been removed. I also moved the logic pertaining to play icons in this file into _videos.scss as Elspeth and I agreed that this was more logical. 

In addition, I have refactored the logic for custom play buttons so that it responds to a screen resizing event listener as well as on page load. 

I have tested this fix using Browserstack on both android and iOs devices (new and old) without issues. Screenshot here showing android play icon displaying correctly: 

Screen Shot 2019-03-26 at 11 27 50 AM

### CODE REVIEW

**Comment by l-suzuki** [comment]
@elspeth-rabid I tested this on Browserstack using iPhone X and it worked fine, so I am assuming the breakpoint 'large' is still ok because of this? But lmk if you think otherwise :)

File: app/assets/javascripts/custom_play_button.js [hunk 1]
```diff
@@ -1,41 +1,59 @@
...
   });
 
-  $('.play-button').click(function(e) {
+  $(".play-button").click(function(e) {
     e.preventDefault();
     videoResponse(this.nextElementSibling);
   });
 
+  function adjustVideoControlsToScreenSize() {
+    if (Modernizr.touch && !Foundation.MediaQuery.atLeast("large")) {
```
Comment (elspeth-rabid) [inline app/assets/javascripts/custom_play_button.js:21]: There needs to be an else to this situation where you show the play button and hide the controls.

File: app/assets/stylesheets/_buttons.scss [hunk 2]
```diff
@@ -119,17 +128,3 @@ a {
   height: 1.5rem;
   margin-left: 0;
 }
-
```
Comment (elspeth-rabid) [inline app/assets/stylesheets/_buttons.scss:122]: ✨

File: app/assets/stylesheets/_video.scss [hunk 3]
```diff
@@ -35,3 +35,10 @@ video {
     display: block;
   }
 }
+
+// Hides the default videojs BigPlayButton because we have our own custom one
+.video-js {
```
Comment (elspeth-rabid) [inline app/assets/stylesheets/_video.scss:40]: ✨

**Review by elspeth-rabid — CHANGES_REQUESTED** [review]
Just the linting and the else case :-)

File: app/assets/javascripts/custom_play_button.js (hunk 1 above)
Comment (l-suzuki) [reply app/assets/javascripts/custom_play_button.js:21]: I'm not sure I understand why this is necessary - could you elaborate? :)

Comment (l-suzuki) [reply app/assets/javascripts/custom_play_button.js:21]: As in -- isn't the else case you're describing the default situation? So if the 'if' conditions aren't met it just defaults back to that anyway?

**Pull request closed without merging**